awswrangler/data_api/redshift.py

```diff
@@ -54,21 +58,28 @@ def __init__(
         super().__init__(self.client, logger)
 
     def _validate_auth_method(self) -> None:
-        if self.secret_arn == "" and self.db_user == "":
```

Comment:
Should we be checking "truthyness" here? If `secret_arn` is None, this condition wouldn't activate. If this gets written as
```python
if not self.workgroup_name and not self.secret_arn and not self.db_user:
```
it should catch `None` and empty strings.